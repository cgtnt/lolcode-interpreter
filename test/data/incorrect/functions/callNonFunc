HAI

I HAS A ADD ITZ 5

VISIBLE I IZ ADD MKAY

KTHXBYE
