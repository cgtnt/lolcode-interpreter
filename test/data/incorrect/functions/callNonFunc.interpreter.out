Line 5: Line 5: Cannot call ADD, not a function
